index.html

```diff
@@ -75,69 +84,75 @@ <h4 class="about-head">Tonic</h4>
         </p>
         <ul class="tech1">
           <li class="tech-t">HTML</li>
+
           <li class="tech-t">CSS</li>
           <li class="tech-j">JavaScript</li>
         </ul>
         <button type="button" class="project-btn">See Project</button>
       </div>
     </div>
+    </div>
     <div class="about-section">
-      <img src="./img/Snapshoot Portfolio (1).png" class="card-img" />
-      <h4 class="about-head">Multi-Post <br> Stories</h4>
+      <img src="./img/Dbg2.png" class="card-img" />
       <div class="canopy">
+        <h4 class="about-head">Multi-Post Stories</h4>
         <ul class="cano">
-          <li class="tec">CANOPY</li>
-          <li class="tech">Back End Dev</li>
+          <li class="tec">FACEBOOK</li>
+          <li class="tech">Full Stack Dev</li>
           <li class="tech">2015</li>
         </ul>
         <p class="canopy-txt">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required
+          Experimental content creation feature that allows users to add to an existing story over 
+          the course of a day without spamming their friends.
         </p>
         <ul class="tech1">
           <li class="tech-t">HTML</li>
+          <li class="tech-r">Ruby On Rails</li>
           <li class="tech-t">CSS</li>
           <li class="tech-j">JavaScript</li>
         </ul>
```

Comment:
[OPTIONAL] Please ensure that you give some breathing space in all your  `tech1` class items, below is an Illustration between yours and the given design. :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Figma Design</td>
    </tr>
    <tr>
      <td>

![S design 2](https://github.com/Danny-imo/my-project/assets/43528594/fa471aad-95c9-4727-bf49-dad77e497fc7)

</td>
      <td>

![Figma Design 2](https://github.com/Danny-imo/my-project/assets/43528594/5b3de4dc-efb7-4998-ac4d-5ee2bd980914)

</td>
    </tr>
  </table>